examples/expression-oriented-style/expression_oriented_style.bal

```diff
@@ -0,0 +1,30 @@
+import ballerina/io;
+
+// You can define the return value expression using the `=>` notation instead of using
+// curly braces to define the function body block when the function body is just
+// a return statement with an expression.
+function inc1(int x) returns int => x + 1;
+
+// `inc2(int x)` is the same as` inc1(int x)`.
+function inc2(int x) returns int {
+    return x + 1;
+}
+
+var obj = object {
+    private int x = 1;
+
+    function getX() returns int => self.x;
+};
+
+function hypot(float x) returns float =>
+    let float x2 = x * x in float:sqrt(x2 + x2);
```

Comment:
It is better to add a comment for things we want to emphasize in a BBE. Or the user may miss the application.